frontend/kalunwa-web-app/src/app/features/components/contact-us/contact-us.component.html

```diff
@@ -0,0 +1,73 @@
+<div>
+    <div class = "header" id="top">
+        <img src="assets/images/carousel/carousel1.jpg"/>
+    </div>
+  
+    <div class ="subhead">
+        <h1>Contact Us</h1>
+        <p>Send us a message!</p>
+    </div>
+</div>
+<div class="form">
+
+    <form #form action="https://formsubmit.co/firecav198@bunlets.com" method="POST">
+        <div class="row1">            
+            <label for="pronoun">Pronoun
+                <br>
+                <select name="pronoun" id="pronoun">
+                    <option value="Mr."> Mr.</option>
+                    <option value="Ms.">Ms.</option>
+                    <option value="Mx.">Mx.</option>
+                </select>
+            </label>
+            
+            <label for="name">Name
+                <br>
+                <input type="text" name="name" id="name" placeholder="Jose Rizal" required>
+            </label>
+        </div>
+        
+        <div class="row2">
+            <label for="email">Email
+                <br>
+                <input type="email" name="email" id="email" placeholder="j.riz@gmail.com" required>
+            </label>
+        </div>
+        
+        <div class="row3">
+            <label for="message">Message
+                <br>
+                <textarea name="message" id="message" placeholder="Your Message Here" required></textarea>
+            </label>
+        </div>
+        <button type="submit" class="btn btn-success" (click)="form.submit()">Submit</button>
```

Comment:
E change lang guro natu iyang color to dark green? nya dle sd ka mag mat-button ani po?